internal/databases/mongo/oplog/applier.go

```diff
@@ -123,13 +123,13 @@ func (ap *DBApplier) Close(ctx context.Context) error {
 	return nil
 }
 
-func (ap *DBApplier) shouldSkip(op, ns string) error {
-	if op == "n" {
+func (ap *DBApplier) shouldSkip(oplog *db.Oplog) error {
+	if oplog.Namespace == "n" {
 		return fmt.Errorf("noop op")
 	}
 
-	// sharded clusters are not supported yet
-	if strings.HasPrefix(ns, "config.") {
+	//sharded clusters are not supported yet
```

Comment:
PR name says, that we are starting support of restore of sharded clusters, but comment says, that we don't support it yet. Looks a bit weird

Reply:
fixed